sopel/module.py

```diff
@@ -8,4 +8,38 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from sopel.plugin import *  # noqa
+# Import everything from sopel.plugin at the time of replacement.
+# Everything new from this point on must *not* leak here.
+# Therefore, don't add anything to this import list. Ever.
+from sopel.plugin import (  # noqa
+    ADMIN,
+    action_commands, commands,
+    echo,
+    event,
+    example,
+    find,
+    HALFOP,
+    intent,
+    interval,
+    label,
```

Comment:
`label` wasn't available in any release yet, and shouldn't be included here.

```suggestion
```

Reply:
Hm... yeah, I'll need to change the tests accordingly then.